camel/retrievers/bm25_retriever.py

```diff
@@ -33,8 +33,8 @@ class BM25Retriever(BaseRetriever):
             calculating document scores.
         content_input_path (str): The path to the content that has been
             processed and stored.
-        chunks (List[Any]): A list of document chunks processed from the
-            input content.
+        unstructured_modules (UnstructuredIO): A module for parsing files and
+            URLs and chunking content based on specified parameters.
 
```

Comment:
I've noticed that the number of characters in certain lines exceeds eighty in different places. Have we updated any new coding standards, Ruff?